### PR TITLE
d2 0.2.3

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  err: failed to compile: open /private/tmp/d2-test-20230304-2027-1qvus9y/private/tmp/d2-test-20230304-2027-1qvus9y/test.d2: no such file or directory

```

**Review by SMillerDev — APPROVED** [review]

File: Formula/d2.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def install
       homebrew-core -> brew: depends
     EOS
 
-    system bin/"d2", test_file
+    system bin/"d2", "test.d2"
```
Comment (alixander) [inline Formula/d2.rb:33]: @Porkepix why did this change in the bump?

Comment (alixander) [reply Formula/d2.rb:33]: hm strange, we'll look into it. thank you for bumping 🙏

Comment (Porkepix) [reply Formula/d2.rb:33]: It wasn't changed by the bump but by @chenrui333 who did this to fix the broken tests.

Comment (nhooyr) [reply Formula/d2.rb:33]: Fixed this bug in master so next release this can be reverted.

Comment (nhooyr) [reply Formula/d2.rb:33]: https://github.com/terrastruct/d2/pull/979

Comment (alixander) [reply Formula/d2.rb:33]: it'll continue to work without reverting anyway right

Comment (nhooyr) [reply Formula/d2.rb:33]: Yup, but I believe the convention is to use `test_file` and not hard code the path. It doesn't matter much though yes.

**Pull request closed without merging**